tests/e2e/network/ensureloadbalancer.go

```diff
@@ -88,6 +88,47 @@ var _ = Describe("Ensure LoadBalancer", func() {
 		tc = nil
 	})
 
+	FIt("should support mixed protocol services", func() {
```

Comment:
do we need to enable feature gates on capz and aks-engine cluster configs?

Reply:
aks-engine enabled in https://github.com/kubernetes-sigs/cloud-provider-azure/pull/898
capz pending for https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1847